engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/UpdateByBucketHelper.java

```diff
@@ -368,10 +378,7 @@ public void processWindow(final int winIdx, final boolean initialStep) {
      * Close the window contexts and release resources for this bucket
      */
     public void finalizeUpdate() {
-        for (int winIdx = 0; winIdx < windows.length; winIdx++) {
-            windowContexts[winIdx].close();
-            windowContexts[winIdx] = null;
-        }
+        SafeCloseableArray.close(windowContexts);
```

Comment:
We should standardize on this. In a lot of places you are instead using `SafeCloseable.closeArray` followed by an `Arrays.fill`, and this is much better actually.